packages/@react-aria/dnd/stories/DroppableGrid.tsx

```diff
@@ -135,6 +140,13 @@ function DroppableGrid(props) {
     })
   });
 
+  React.useImperativeHandle(ref, () => ({
```

Comment:
why not destructure the import useImperativeHandle like we do for other React hooks?

Reply:
🤷 it's just an example. Several other places in this file are like this.

Reply:
ah, yep, didn't notice it was in an example